content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -135,18 +135,18 @@ Iterating through a dictionary representation of an ordered set:
 nums = {0: None, 1: None, 2: None, 4: None, 7: None}
 iterator = iter(nums)
 
-print(next(iterator)) # 0
-print(next(iterator)) # 1
-print(next(iterator)) # 2
-print(next(iterator)) # 4
-print(next(iterator)) # 7
+print(next(iterator))  # 0
```

Comment:
lmao bro how do you manage to revert these every time

Reply:
I keep forgetting to update it in the editor lol